doc/changelog.md

```diff
@@ -67,132 +67,141 @@
-  (Array([1., 0.], dtype=float64), Array([1., 0.], dtype=float64), Array([1., 0.], dtype=float64), Array([0.5, 0.5], dtype=float64))
+  Note that setting the seed will *not* avoid shot-noise stochasticity in terminal measurement
+  statistics such as `sample` or `expval`:
 
+  ```python
+  dev = qml.device("lightning.qubit", wires=1, shots=10)
+
+  @qml.qnode(dev)
+  def circuit(x):
+      qml.RX(x, wires=0)
+      m = measure(0)
+
+      if m:
+          qml.Hadamard(0)
+
+      return qml.expval(qml.PauliZ(0))
+
+  @qjit(seed=37, autograph=True)
+  def workflow(x):
+      return jnp.stack([circuit(x) for i in range(4)])
+  ```
+  ```pycon
+  >>> workflow(1.8)
+  Array([1. , 1. , 1. , 0.4], dtype=float64)
+  >>> workflow(1.8)
+  Array([ 1. ,  1. ,  1. , -0.2], dtype=float64)
   ```
 
-* Exponential extrapolation is now a supported method of extrapolation when using `mitigate_with_zne`.
+* Exponential extrapolation is now a supported method of zero-noise extrapolation when performing
+  error mitigation in Catalyst using `mitigate_with_zne`.
   [(#953)](https://github.com/PennyLaneAI/catalyst/pull/953)
 
   This new functionality fits the data from noise-scaled circuits with an exponential function,
-  and returns the zero-noise value. This functionality is available through the pennylane module
-  as follows
+  and returns the zero-noise value:
+
   ```py
   from pennylane.transforms import exponential_extrapolate
+  from catalyst import mitigate_with_zne
 
-  catalyst.mitigate_with_zne(
-      circuit, scale_factors=jax.numpy.array([1, 2, 3]), extrapolate=exponential_extrapolate
-  )
+  dev = qml.device("lightning.qubit", wires=2, shots=100000)
+
+  @qml.qnode(dev)
+  def circuit(weights):
+      qml.StronglyEntanglingLayers(weights, wires=[0, 1])
+      return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
+
+  @qjit
+  def workflow(weights, s):
+      zne_circuit = mitigate_with_zne(circuit, scale_factors=s, extrapolate=exponential_extrapolate)
+      return zne_circuit(weights)
   ```
 
-* A frontend decorator can be applied to a qnode to signal a compiler pass run.
+  ```pycon
+  >>> weights = jnp.ones([3, 2, 3])
+  >>> scale_factors = jnp.array([1, 2, 3])
+  >>> workflow(weights, scale_factors)
+  Array(-0.19946598, dtype=float64)
+  ```
+
+* A new module is available, `catalyst.passes`, which provides Python decorators
+  for enabling and configuring Catalyst MLIR compiler passes.
   [(#911)](https://github.com/PennyLaneAI/catalyst/pull/911)
 
-  A new module, `catalyst.passes` (file `frontend/catalyst/passes.py`), is added to
-  provide UI access for pass decorators. This PR adds the `cancel_inverses` decorator,
-  which runs the `-removed-chained-self-inverse` mlir pass that cancels two neighbouring
-  Hadamard gates.
+  The first pass available is `catalyst.passes.cancel_inverses`,
+  which enables the `-removed-chained-self-inverse` MLIR pass that
+  cancels two neighbouring Hadamard gates.
 
   ```python
```

Comment:
Have simplified the example for the changelog

Reply:
Thanks, this is indeed much cleaner!